lib/rucio/common/utils.py

```diff
@@ -925,10 +923,24 @@ def detect_client_location():
                                          os.environ.get('OSG_SITE_NAME',
                                                         'ROAMING')))
 
+    latitude = os.environ.get('RUCIO_LATITUDE')
+    longitude = os.environ.get('RUCIO_LONGITUDE')
```

Comment:
I wonder if it makes sense to add `rucio.cfg` settings for these

Reply:
That was my proposal too, but Martin preferred this.